config/config.go

```diff
@@ -282,6 +282,7 @@ type TLS struct {
 	PreferServerCipherSuites bool     `mapstructure:"prefer_server_cipher_suites"`
 	CipherSuites             []uint16 `mapstructure:"cipher_suites"`
 	EnableMTLS               bool     `mapstructure:"enable_mtls"`
+	DisableSystemCaPool      bool     `mapstructure:"disable_system_ca_pool"`
```

Comment:
this property should also be declared at https://github.com/luraproject/lura/blob/master/config/parser.go#L221 and initiated at https://github.com/luraproject/lura/blob/master/config/parser.go#L193